### PR TITLE
added error message to snapshot modal if snapshot is incorrect

### PR DESCRIPTION
**Pull request by Israellund**
## Link to Issue
Closes: #6314 

## Description of Changes
-Added a new error message to Snapshot modal if snapshot address is incorrect

## "How We Fixed It"

-Used a setTimeout to show the error message after 6 seconds if the cow is still showing

## Test Plan
-go to Integration in Admin Capabilities and input a nonsense snapshot address
-now create a thread or find one that doesn't have a snapshot connected to it
-try to connect a snapshot
-notice that after 6 seconds an error message is shown. 

## Other Considerations

https://github.com/hicommonwealth/commonwealth/assets/69872984/7c1b327f-3e4d-4962-85f2-fd36a696a039

-

### CODE REVIEW

**Review by jnaviask — CHANGES_REQUESTED** [review]
Doesn't seem to work for me... why do we need a timeout rather than being able to catch an error immediately + then present a toast?

File: packages/commonwealth/client/scripts/views/pages/new_snapshot_proposal/index.tsx [hunk 1]
```diff
@@ -92,6 +93,15 @@ export const NewSnapshotProposalForm = ({
     }
   };
 
+  useEffect(() => {
+    const errorTimeout = setTimeout(() => {
```
Comment (mzparacha) [inline packages/commonwealth/client/scripts/views/pages/new_snapshot_proposal/index.tsx:97]: why are we using timeout here?

**Comment by Israellund** [comment]
Changed the logic to not use a setTimeout but instead to look for if a space === null. 

https://github.com/hicommonwealth/commonwealth/assets/69872984/9dc38656-f15d-4f2c-b244-e8b0933ec7fb

**Comment by Israellund** [comment]
I am closing this PR due to unresolvable merge conflicts, as the ticket was originally started a while ago. A new PR off of the newest master can be found at #6928

**Comment by Israellund** [comment]
I am closing this PR due to unresolvable merge conflicts, as the ticket was originally started a while ago. A new PR off of the newest master can be found at https://github.com/hicommonwealth/commonwealth/pull/6928

**Comment by Israellund** [comment]
I am closing this PR due to unresolvable merge conflicts, as the ticket was originally started a while ago. A new PR off of the newest master can be found at https://github.com/hicommonwealth/commonwealth/pull/6928

**Pull request closed without merging**